packages/common/core-interfaces/Removing-IFluidRouter.md

```diff
@@ -90,7 +90,36 @@ const entryPoint = await container.getEntryPoint();
 
 ### Aliased DataStores
 
-(Not yet written)
+The `getRootDataStore` API has been deprecated on `IContainerRuntime` and `ContainerRuntime`. Migrate to usage of the new `IContainerRuntime.getAliasedDataStoreEntryPoint(...)` endpoint.
+The following is an example of that this change may look like:
+
+```ts
+// OLD
+const alias = "alias";
+const dataStore = await containerRuntime.createDataStore(TestDataObjectType);
+await dataStore.trySetAlias(alias);
+
+const retrievedDataStore = await containerRuntime.getRootDataStore(alias);
+```
+
+```ts
+// NEW
+const alias = "alias";
+const dataStore = await containerRuntime.createDataStore(TestDataObjectType);
+await dataStore.trySetAlias(alias);
+
+const entryPoint: IFluidHandle<ITestDataObject> | undefined =
+	await containerRuntime.getRootDataStore(alias);
```

Comment:
@kian-thompson is this supposed to use `getAliasedDataStoreEntryPoint`?

Reply:
You're correct. PR to fix the note: https://github.com/microsoft/FluidFramework/pull/19581